src/actions/types.js

```diff
@@ -1,3 +1,5 @@
-export const FETCH_RESTAURANTS_SUCCESS = 'FETCH_RESTAURANTS_SUCCESS';
-export const FETCH_RESTAURANTS_FAILURE = 'FETCH_RESTAURANTS_FAILURE';
-export const FETCH_RESTAURANTS_STARTED = 'FETCH_RESTAURANTS_STARTED';
\ No newline at end of file
+export const FETCH_RESTAURANTS_SUCCESS  = 'FETCH_RESTAURANTS_SUCCESS';
+export const FETCH_RESTAURANTS_FAILURE  = 'FETCH_RESTAURANTS_FAILURE';
+export const FETCH_RESTAURANTS_STARTED  = 'FETCH_RESTAURANTS_STARTED';
+export const ADD_RESTAURANT_SUCCESS     = 'ADD_RESTAURANT_SUCCESS';
```

Comment:
You might want to handle the "api in progress" state while a new restaurant is attempted to be added.